www/annotationdb/annotationdb.css

```diff
@@ -0,0 +1,443 @@
+div.wy-nav-content{
```

Comment:
OK, I'm now shrinking the padding on the right rather than adjusting the max-width, so I think the left margin should align now. Though I think it might not be a bad idea to shrink that left margin on all of the pages.